src/commands/synthetics/tunnel.ts

```diff
@@ -142,7 +143,11 @@ export class Tunnel {
     }
 
     // Username is allowed and key authentication was successful
-    this.log(`Proxy opened for test ${user}`)
+    if (!this.openedTunnels.has(ctx.username)) {
+      // Limit to one log per test
+      this.openedTunnels.add(ctx.username)
+      this.log(`Proxy opened for test ${ctx.username}`)
+    }
```

Comment:
Nit, maybe removing the `ctx.username` at the end of the life-cycle of the tunnel?
Another solution would be to throttle based on the username?

Reply:
The life-cycle of the tunnel is basically the process itself, there won't be more than one opened in the command life-cycle so I think we're good to leave them there

Reply:
These logs will not bring much value to an user at a test level (it is unlikely that one test amongst a suite will not have its tunnel opened), could we make it be displayed only once or at most once per tunnel?